Makefile

```diff
@@ -68,6 +82,11 @@ bin/k8s:
 bin/unpack:
 	CGO_ENABLED=0 go build -o $@ ./cmd/unpack/...
 
+build-local-container: ## Builds provisioner container image locally
+	docker build -f Dockerfile -t $(IMAGE) .
+
+load-image: ## Load-image loads the currently constructed image onto the cluster
+	 kind load docker-image $(IMAGE)
```

Comment:
It would be nice if we outlined these steps in dev documentation at some point.

Reply:
Combo has an outline for this we could just template from.

Reply:
That sounds good

> Can we separate out the Makefile changes into their own commit?

Done